### PR TITLE
Peer20-310 Change post's type on Post Page

### PR DESCRIPTION
**Pull request by Dmitrijg24k**

### CODE REVIEW

File: app/containers/ViewQuestion/ContentHeader.jsx [hunk 1]
```diff
@@ -168,7 +168,7 @@ const ContentHeader = props => {
           {type === QUESTION_TYPE && (
             <Button
               id={`${type}_change_type_with_rating_restore_${answerId}`}
-              show={isGlobalAdmin || isChangeTypeAvailable}
+              show={(isGlobalAdmin || isChangeTypeAvailable) && questionData.postType !== 2}
```
Comment (sergeyi23) [inline app/containers/ViewQuestion/ContentHeader.jsx:171]: Use constant `POST_TYPE.tutorial`

File: app/containers/ViewQuestion/saga.js [hunk 2]
```diff
@@ -143,8 +143,7 @@ import orderBy from 'lodash/orderBy';
 
 export const isGeneralQuestion = question => Boolean(question.postType === 1);
 
-export const getQuestionTypeValue = isGeneral =>
-  isGeneral ? QUESTION_TYPES.GENERAL.value : QUESTION_TYPES.EXPERT.value;
+export const getQuestionTypeValue = PostType => PostType === 1 ? 0 : 1;
```
Comment (sergeyi23) [inline app/containers/ViewQuestion/saga.js:146]: Use existing constants instead of hardcoding values

File: app/containers/ViewQuestion/saga.js [hunk 3]
```diff
@@ -162,7 +161,6 @@ export function* getQuestionData({
   } else {
     question = yield call(getQuestionFromGraph, +questionId);
     question.commentCount = question.comments.length;
-    question.communityId = Number(question.communityId);
```
Comment (sergeyi23) [inline app/containers/ViewQuestion/saga.js:165]: Why was this line removed?

File: app/containers/ViewQuestion/saga.js [hunk 4]
```diff
@@ -143,8 +143,8 @@ import orderBy from 'lodash/orderBy';
 
 export const isGeneralQuestion = question => Boolean(question.postType === 1);
 
-export const getQuestionTypeValue = isGeneral =>
-  isGeneral ? QUESTION_TYPES.GENERAL.value : QUESTION_TYPES.EXPERT.value;
+export const getQuestionTypeValue = PostType =>
```
Comment (ruslanzharkov) [inline app/containers/ViewQuestion/saga.js:146]:
```suggestion
export const getQuestionType = postType => 
```

File: app/containers/ViewQuestion/saga.js [hunk 5]
```diff
@@ -143,8 +143,8 @@ import orderBy from 'lodash/orderBy';
 
 export const isGeneralQuestion = question => Boolean(question.postType === 1);
 
-export const getQuestionTypeValue = isGeneral =>
-  isGeneral ? QUESTION_TYPES.GENERAL.value : QUESTION_TYPES.EXPERT.value;
+export const getQuestionTypeValue = PostType => 
+  PostType === POST_TYPE.generalPost ? POST_TYPE.expertPost : POST_TYPE.generalPost;
```
Comment (ruslanzharkov) [inline app/containers/ViewQuestion/saga.js:147]:
```suggestion
  postType === POST_TYPE.generalPost ? POST_TYPE.expertPost : POST_TYPE.generalPost;
```

**Review by ruslanzharkov — APPROVED** [review]
